### PR TITLE
Update docs for new micro-deposits endpoints

### PR DESCRIPTION
**Pull request by johndburk**
add docs for:
* /plaid_webhook_auth - Automated micro-deposit
* /plaid_sameday_auth - Same Day micro-deposit auth phase II

NEW: added Docker compose files for running Slate server in a container,
no need to install Ruby and associated bundles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <p align="center">
   <img src="https://raw.githubusercontent.com/lord/img/master/logo-slate.png" alt="Slate: API Documentation Generator" width="226">
   <br>
-  <a href="https://travis-ci.org/lord/slate"><img src="https://travis-ci.org/lord/slate.svg?branch=master" alt="Build Status"></a>
+  <!---
```
Comment (johndburk) [inline README.md]: Commented out due to it crashing any Jetbrains IDE (pycharm, webstorm, etc)

**Comment by rockonjohn** [comment]
@johndburk - can you add some info in /link_account adding the info about configuring for microdeposits? Something along the lines of: "If you are enabling Microdeposits for 100% coverage of all banks, you will also need to include the following in your frontend Plaid configuration:..."

**Comment by rockonjohn** [comment]
JB, some edits in addition to those commented in the code:

- Change all instances of micro-deposit(s) to microdeposit
- Update the /link_account endpoint documentation with:
`At a minimum, your frontend Plaid Link integration will need to: 

- use a public key from Sila. For sandbox apps, please use the public key: **"fa9dd19eb40982275785b09760ab79"**
- be set to the **"sandbox"** environment for sandbox apps, and the **"production"** environment for production apps
- use the **"auth"** product from Plaid
**NOTE:** For 100% coverage of all supported institutions, we strongly recommend passing additional parameters when creating the Plaid Link interface to enable microdeposit verification. This configuration is provided in detail [here.](#same-day-micro-deposit-authentication)`
(head's up - the link will probably change to #same-day-micro-deposit-authentication when you make the 1st update.

Thanks!

File: source/includes/_plaid-microdeposits.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+          }).catch((err) => {
+            console.log('  ... looks like we ran into an issue!')
+            this.handleError(err)
+          })
+        }
+    })
+```
+Once the user has noted the amounts of the two micro-deposits in their account, your client-side application should 
+call the Sila API [/plaid_sameday_auth](#plaid_sameday_auth), passing in the `account_name`.  This endpoint's response
+will include a `public_token` which is then used to initialize the Plaid Link integration in **update** mode. (also
```
Comment (rockonjohn) [inline source/includes/_plaid-microdeposits.md]: **update** mode through which your end-user can confirm the deposit amounts made in their account.

**Comment by johndburk** [comment]
> JB, some edits in addition to those commented in the code:
> 
> * Change all instances of micro-deposit(s) to microdeposit
> * Update the /link_account endpoint documentation with:
>   `At a minimum, your frontend Plaid Link integration will need to:
> * use a public key from Sila. For sandbox apps, please use the public key: **"fa9dd19eb40982275785b09760ab79"**
> * be set to the **"sandbox"** environment for sandbox apps, and the **"production"** environment for production apps
> * use the **"auth"** product from Plaid
>   **NOTE:** For 100% coverage of all supported institutions, we strongly recommend passing additional parameters when creating the Plaid Link interface to enable microdeposit verification. This configuration is provided in detail [here.](#same-day-micro-deposit-authentication)`
>   (head's up - the link will probably change to #same-day-micro-deposit-authentication when you make the 1st update.
> 
> Thanks!

Think I've got your requested changes in the correct spots, had to sort out what was just context in the comment and what was a change.  Let me know if I've missed anything.

**Pull request merged**